sfdx-source/apex-mocks/main/default/classes/fflib_QualifiedMethodAndArgValuesTest.cls

```diff
@@ -0,0 +1,24 @@
+@isTest
+public with sharing class fflib_QualifiedMethodAndArgValuesTest
```

Comment:
`private class ...`?

Reply:
I agree @stohn777.  There is no need to have `with sharing` in a test class

Reply:
OK, yeah agreed. I was just going off of other tests in the codebase that had it (https://github.com/apex-enterprise-patterns/fflib-apex-common/blob/master/fflib/src/classes/fflib_SObjectDomainTest.cls, for example).